ui/components/ui/contract-token-values/index.scss

```diff
@@ -0,0 +1,26 @@
+.contract-token-values {
+  align-items: center;
+  justify-content: center;
```

Comment:
We have props o nthe box component that apply these

Reply:
Hi @brad-decker, Done [here](https://github.com/MetaMask/metamask-extension/pull/15617/commits/1851df15766128331cb4a3d334d3189798f17c66)